web/src/components/tasks/LicenseTask.test.tsx

```diff
@@ -450,43 +445,17 @@ describe("<LicenseTask />", () => {
 
       it("displays NOT_FOUND error alert when license status cannot be found", async () => {
         useMockBusiness({
-          licenseData: generateLicenseData(
-            {},
-            {
-              [taskIdLicenseNameMapping[task.id]]: generateLicenseDetails({
-                lastUpdatedISO: "",
-              }),
-            }
-          ),
+          licenseData: { lastUpdatedISO: "", licenses: {} },
```

Comment:
Pebble - it would make more sense to provide license data for tasks that are not relevant to the page and confirm the alert appears. The idea is other data can be retrieved and the alert should appear for tasks that are not relevant. It you keep the existing changes then further tests are needed.